File: src/main/java/com/example/app/utils/JwtUtils.java
package com.example.app.utils;

import io.jsonwebtoken.Claims;

public class JwtUtils {

    public static JwtAuthentication generate(Claims claims) {
        final JwtAuthentication jwtInfoToken = new JwtAuthentication();
        jwtInfoToken.setLogin(claims.getSubject());
        return jwtInfoToken;
    }
}

File: src/main/kotlin/com/example/utils/JwtUtils.kt
package com.example.utils

import com.example.utils.security.JwtAuthentication
import io.jsonwebtoken.Claims

object JwtUtils {
    fun generate(claims: Claims): JwtAuthentication {
        val jwtInfoToken = JwtAuthentication()
        jwtInfoToken.setLogin(claims.subject)
        return jwtInfoToken
    }
}
